spec/features/course/assessment/submissions_viewing_spec.rb

```diff
@@ -72,23 +72,17 @@
     context 'As a Course Student' do
       let(:user) { create(:course_student, course: course).user }
 
-      scenario 'I can view my submitted and graded submissions' do
```

Comment:
The test is trying to check if students can see the submissions of a certain status. If introducing the unique_index clashes with `attempting_submission`, `submitted_submission` and `graded_submission`, apologies as I created 3 submissions for a student with 1 assessment. 

I would think the way to modify is to then have 1 user, 3 assessments, each with 1 submission (total of 3, one for each different status). Would that work? @allenwq @fonglh